test/clientTest.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+from past.builtins import basestring
+import unittest
+import time
+import os
+import subprocess
+import signal
+import requests
+
+
+class ClientTest(unittest.TestCase):
+    """A set of test cases for the wes-client."""
+    def setUp(self):
+        """Start a (local) wes-service server to make requests against."""
+        self.wes_server_process = subprocess.Popen('python {} --debug'.format(os.path.abspath('wes_service/wes_service_main.py')), shell=True)
```

Comment:
Suggest piping the output as it makes for a noisy test: https://github.com/ga4gh/data-object-service-schemas/blob/master/python/test/test_server.py#L24

Reply:
Done and done.  Though oddly I had to pipe the subprocess `stderr` as well to kill the log `INFO` messages.